src/pages/overlay/components/overlay/Overlay.tsx

```diff
@@ -11,6 +11,10 @@ import styles from './overlay.module.css'
 
 interface OverlayProps {
     sleeping: boolean,
+    awoken: {
```

Comment:
It essentially acts as an event emitter, so that the overlay can know when the user is active which stops the automatic dismissal happening for the chat commands.